Sources/Common/Core/HalfFloat/index.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Helper class to convert to/from half float
+ */
+
+/**
+ * Convert a Base64 string to an ArrayBuffer.
```

Comment:
Is that the real description?

Reply:
See that is why I have you review it :-) But really I was worried about the types I chose to use.

Reply:
number is fine, but now, I just noticed that you export both function as default. 

That is wrong. You should export your function as-is (no default keyword).

Then add at the bottom 

```
export default {
  toHalf,
  fromHalf,
};
```

Reply:
By doing that also in your JS you should export your function as well. That would allow someone to import them individually to match the TS description.

Reply:
or maybe
```
export declare const HalfFloat: {
	toHalf: typeof toHalf;
	fromHalf: typeof fromHalf;
}

export default HalfFloat;
```

Reply:
Does that look OK?

Reply:
yes it is.